schema/constants.go

```diff
@@ -36,6 +36,7 @@ const (
 	CreateTableCouponsSQL                   = "create table coupons (slug text, code text, hash text);"
 	CreateIndexCouponsSQL                   = "create index index_coupons on coupons (slug);"
 	CreateTableModeratedStoresSQL           = "create table moderatedstores (peerID text primary key not null);"
+	CreateOrderMessagesSQL                  = "create table order_messages (messageID text primary key not null, orderID text, message_type integer, message blob, peerID text, url text, acknowledged bool, tries integer, created_at integer, updated_at integer);"
 	// End SQL Statements
```

Comment:
We should consider adding indices for the `Messages` table and using normalized data for the lookup. (ex: `where peerID = ? and orderType = ?`.

Reply:
Three indices have been added: MessageID, <OrderID, MessageType> and <PeerID, MessageType>